rllib/connectors/common/add_states_from_episodes_to_batch.py

```diff
@@ -335,6 +346,17 @@ def __call__(
                     num_items=int(math.ceil(len(sa_episode) / max_seq_len)),
                     single_agent_episode=sa_episode,
                 )
+                if Columns.NEXT_OBS in batch:
+                    self.add_n_batch_items(
+                        batch=batch,
+                        column="new_state_in",
```

Comment:
nit: create a new Column constant?

Reply:
I had a similar idea, but did not before it was running. Have created a new column now.